Observer/ImportCategories.php

```diff
@@ -122,7 +131,16 @@ private function _getCategoryJson()
     private function _importCategories()
     {
         $categoryJson = $this->_getCategoryJson();
-        $this->resourceConfig->saveConfig(TaxjarConfig::TAXJAR_CATEGORIES, json_encode($categoryJson), 'default', 0);
-        $this->reinitableConfig->reinit();
+
+        foreach ($categoryJson as $categoryData) {
+            $category = $this->categoryFactory->create();
+
+            $this->categoryResourceModel->load($category, $categoryData['product_tax_code'], 'product_tax_code');
```

Comment:
@ryanreeves-taxjar Can you add a brief comment above here explaining that we're loading by `product_tax_code` to prevent duplicates?